### DISCUSSION TITLE
UK Mobile ecosystems market study

### DISCUSSION THREAD

**Discussion by daKmoR** [modernweb-dev/web#1871] [General]
The UK is currently doing an [official market study](https://www.gov.uk/cma-cases/mobile-ecosystems-market-study) around the mobile ecosystem.

On the 26th of January 2022 an [interim report](https://www.gov.uk/government/publications/mobile-ecosystems-market-study-interim-report/interim-report) was published.

Especially the section "[5. Competition in the supply of mobile browsers](https://www.gov.uk/government/publications/mobile-ecosystems-market-study-interim-report/interim-report#competition-in-the-supply-of-mobile-browsers)" seems very related to web development.

> On iOS devices, Apple bans the use of alternative browser engines – this means that Apple has a monopoly over the supply of browser engines on iOS. It also chooses not to implement – or substantially delays – a wide range of features in its browser engine. This restriction has 2 main effects:
> - limiting rival browsers’ ability to differentiate themselves from Safari on factors such as speed and functionality, meaning that Safari faces less competition from other browsers than it otherwise could do; and
> - limiting the functionality of web apps – which could be an alternative to native apps as a means for mobile device users to access online content – and thereby limits the constraint from web apps on native apps. We have not seen compelling evidence that suggests Apple’s ban on alternative browser engines is justified on security grounds.

Other resources on the topic:
- https://adactio.com/journal/18819
- https://brucelawson.co.uk/2022/one-week-left-to-save-the-web/

## What can you do as an individual to help?

The study is still open for feedback. 
Write a free format email to [mobileecosystems@cma.gov.uk](mailto:mobileecosystems@cma.gov.uk).

Here is once possible text template:

```
Hello,

This response is regarding the "Mobile ecosystems market study" and it contains no confidential information.
[You may attribute this response to my name]
[You shall treat this response as anonymous]

I am [firstname] [lastname] and I am a web developer for [xx] years.

I am often tasked to create an app-like cross-platform experience which is currently often impossible due to 
section 2.5.6 in the Apple App Store rules[1]. The reasons for that are multifold but in short Apples Webkit 
browser is not supporting certain features. Due to those missing features, certain functionality like Bluetooth 
controls is not possible via the web on iOS. On other operating systems that is not an issue as you can tell your 
users: "If you want to use our app on a Mac, Linux, Windows you need to use the Edge Browser". On iOS this 
is however not possible.

The result of this practice is that as a web app developer I am not able to serve this market and either need to 
say this is impossible or requires a native iOS App - which comes with a huge price tag.

For that reason, I would prefer if iOS would allow the installation of other browser engines to not be force into
the native application market for features that are possible on the web but not implemented in WebKit.

Kind regards,

[firstname] [lastname]

[1]: https://developer.apple.com/app-store/review/guidelines/#software-requirements
```

Some considerations for your text:

- A brief summary of the interests or organizations you represent
- whether you are providing any material that you consider to be confidential, and explain why this is the case; and if the response contains confidential information, please also provide a non-confidential version of your response alongside it.
The specific ways in which unfair App Store or Browser choice policies have hurt your business (think lost revenue, increased costs, bugs, etc.)
- Specific missing features and bugs that cause your ongoing business harm
- What you would like CMA to do (e.g., alternative browser engines on iOS) and between the web and native (e.g., the inability to offer a lower-cost, higher service web experience vs being forced into app stores)
- If you are an individual (ie you are not representing an organization), please indicate whether you wish for your response to be attributed to you by name or published anonymously

**Comment by mtom55** [comment]
Consider how the ban on third party browsers and lack of integration and feature parity of Web-Apps leads to the following issues:
1. Blocks the web from being a free and open universal application and distribution platform 
2. Shifts funding away from free open ecosystems to closed proprietary ones
3. Ensures that Apple has a no incentive to fund Safari / Webkit Adequately
4. Blocks interoperability between devices
5. Affects functionality outside of iOS (web developers don't want to use features that aren't universally supported)
6. Greatly increases maintenance and development cost so only larger studios can afford to do them (multiple native apps is more expensive than developing a single web app)
7. Leads to iOS exclusives  
8. Places a severe barrier to entry for new phone entrants because they don't have a collection of apps they can rely on 

Safari Issues:
* Bugs / Stability
* Features / Functionality
* Web-App Integration
* Community Engagement (although this has improved a lot in the last few months)

Key Information:
1. Google gives Apple 15b USD per year for search engine placement in Safari.  This means that ensuring that users Safari and not other browsers is critical to Apple's yearly profit and provides great incentive for them to prevent competition
2. Notifications came out in 2009 for native, in 2022 they are still not available on iOS.  Apple only started working on them in September 2022 after regulatory scrutiny.

What do want:
1. Competition - Third party browsers - the only long term solution that will ever work.  All other solutions might just generate temporary improvements.  We can't predict which browser will bring innovation in the future, it's critical they be allowed to compete.
2. Full Integration of Web-Apps on iOS to the same level of native apps
3. Web-Apps to be unrestricted (subject to user/browser controls) in their functionality
